yourls/yourls.php

```diff
@@ -27,15 +28,12 @@ function yourls_uninstall() {
 
 }
 
+function yourls_addon_settings(App $a, $s) {
```

Comment:
Please keep the reference operator in front of `$s`.

Reply:
Good find!